blue_heron/lib/blue_heron/address.ex

```diff
@@ -0,0 +1,91 @@
+defmodule BlueHeron.Address do
+  @moduledoc """
+  Helper struct for working with BLE Addresses.
+  """
+
+  alias BlueHeron.Address
+
+  defstruct string: "00:00:00:00:00:00",
+            binary: <<00, 00, 00, 00, 00, 00>>,
+            integer: 0x000000000000
+
+  defimpl Inspect, for: Address do
+    def inspect(address, _opts) do
+      "#Address<#{address.string}>"
+    end
+  end
+
+  defimpl String.Chars, for: Address do
```

Comment:
```suggestion
  defimpl String.Chars do
```

When you `defimpl` in the module defining the struct, you can technically omit the `for` clause. I personally like this, others may not so more of an _"FYI if you didn't know and might also like it"_ deal..

Reply:
Thanks, i always forget about that